frontend/src/components/order/order.tsx

```diff
@@ -0,0 +1,86 @@
+import type { ClosedOrderType, OpenOrderType } from "../../lib/types";
+import { getFormattedCurrency } from "../../lib/util";
+
+import styles from "./order.module.css";
+
+export function ClosedOrder(props: ClosedOrderType) {
+  return (
+    <tr>
+      <td>{new Date(props.time).toLocaleDateString()}</td>
+      <td>{props.ticket}</td>
+      <td>{props.symbol || "No symbol"}</td>
+      <td>{props.volume}</td>
+      <td>{props.comment || "No comment"}</td>
+      <td>{props.order_type}</td>
+      <td>{getFormattedCurrency(props.profit)}</td>
+    </tr>
+  );
+}
+
+export function ClosedOrderWrapper({ children }: { children: JSX.Element[] }) {
```

Comment:
I assume children here is a list of ClosedOrders. Could we pass them in as ClosedOrder[] and have the name something like closedOrderList?

Reply:
Not sure how to do this in react. 'children' is a special react prop, so I don't think its type can be changed. Do you know?